vercel.json

```
@@ -1788,6 +1788,82 @@
     {
       "source": "/cloud/release-notes/:path*",
       "destination": "https://temporal.io/change-log/product-area/cloud"
+    },
+    {
+      "source": "/dev-guide/typescript/dev-guide/typescript/foundations#register-types",
+      "destination": "/dev-guide/typescript/foundations#register-types"
+    },
+    {
+      "source": "/typescript/versioning",
+      "destination": "/dev-guide/typescript"
+    },
+    {
+      "source": "/java/quick-start",
+      "destination": "/dev-guide/java"
+    },
+    {
+      "source": "/java/how-to-provide-an-authorization-token-in-java",
+      "destination": "/security#authorizer-plugin"
+    },
+    {
+      "source": "/go/side-effect",
+      "destination": "/dev-guide/go/features#side-effects"
+    },
+    {
+      "source": "/go/how-to-develop-a-workflow-definition-in-go",
+      "destination": "/dev-guide/go/foundations#develop-workflows"
+    },
+    {
+      "source": "/tctl-next/workflow#reset",
+      "destination": "/cli"
+    },
+    {
+      "source": "/tctl-next/modifiers#--event-id",
+      "destination": "/cli/cmd-options#event-id"
+    },
+    {
+      "source": "/php/versioning#workflowgetversion",
+      "destination": "/dev-guide/php"
+    },
+    {
+      "source": "/php/activities#local-activity",
+      "destination": "/dev-guide/php/foundations#develop-activities"
+    },
+    {
+      "source": "/kb/cloud/how-to-manage-certificates-in-temporal-cloud#ca-certificates",
+      "destination": "/cloud/how-to-manage-certificates-in-temporal-cloud#ca-certificates"
+    },
+    {
+      "source": "/kb/additional-resources",
+      "destination": "/"
```

```suggestion
      "destination": "/kb/how-to-explain-temporal"
```
At the moment, this is the closest equivalent, methinks. (We'll need to update this and the next two redirects after we refactor the KB content, but that's a later thing.)